app/javascript/components/pages/Jobs.jsx

```diff
@@ -14,6 +15,8 @@ import 'stylesheets/page';
 import 'stylesheets/jobs';
 
 const Jobs = () => {
+    // eslint-disable-next-line no-unused-vars
+    const [loading, setLoading] = useState(true);
```

Comment:
This sets the `loading` state to true until the jobs are fetched from the backend, at which point loading becomes false and the jobs are displayed.

Reply:
is the linter still complaining even though `loading` is called on L56? 🤔

Reply:
Nope I deleted this line and never committed :) Thanks for catching!